operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -414,19 +413,64 @@ DomainSpec withFluentdConfiguration(boolean watchIntrospectorLog, String credent
       + "environment variables, additional Pod content, and the ability to explicitly start, stop, or restart "
       + "cluster members. The `clusterName` field of each entry must match a cluster that already exists in the "
       + "WebLogic domain configuration.")
-  protected final List<Cluster> clusters = new ArrayList<>();
+  protected final List<ClusterSpec> clusters = new ArrayList<>();
```

Comment:
What is this used for? Will it end up in the schema?

Reply:
I've not  removed the Clusters configuration from the DomainSpec as much of our unit tests are still relying on it.  I didn't modify the unit tests yet as I had just moved most of the effective configuration from the Domain to DomainPresenceInfo and those unit test are validating that the move was fine.